### ISSUE TITLE
Report Webkit / Safari bugs found during cookie banner accessibility testing

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#2135]
## What

Further investigate and report macOS Webkit / Safari bugs found whilst doing accessibility testing for the cookie banner:

- "After pressing 'Hide this page', pressing tab causes the page to reload with "This web page was reloaded because a problem occurred".
- The "This web page was reloaded because a problem occurred" message is not made available to VoiceOver users, and the control to dismiss the message cannot be focused.

## Why

Reporting bugs drives improvements in the web platform, ultimately making things easier for future us and providing a better experience for our users.

## Who needs to know about this

Developers, maybe technical writer?

## Done when

- [x] Minimum test cases created
- [x] Bugs written up and reported
- [x] Bugs tracked in alphagov/reported-bugs
